src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -688,7 +688,10 @@ ApiGlobe.prototype.pan = function pan(pVector) {
  * @return     {number}  The zoom level.
  */
 ApiGlobe.prototype.getZoomLevel = function getZoomLevel() {
-    return this.scene.getMap().getZoomLevel();
+    const theoricalZoom = this.scene.getMap().computeTheoricalZoom(this.scene.currentCamera(), this.getRange());
+    // const idNode = this.scene.gfxEngine.screenCoordsToNodeId();
+    // const realZoom = this.scene.getMap().getZoomLevelFromIdnode(idNode);
+    return theoricalZoom;
```

Comment:
The getZoomLevel method works well when we move the view with the mouse and keyboard controls.
However, I see 2 points :
- when I do a setZoomLevel(X) and then a getZoomLevel(), it does not always return X, but sometimes X+1 (ex : setZoomLevel(1) -> getZoomLevel() = 2

- I think the zoom levels should begin to 0 and finish to 21, ie  minZoomLevel should be 0 instead of 1.